buildSrc/src/main/java/org/elasticsearch/gradle/vagrant/VagrantMachine.java

```diff
@@ -69,7 +73,7 @@ public void execute(Action<VagrantExecSpec> action) {
 
         Objects.requireNonNull(vagrantSpec.command);
 
-        LoggedExec.exec(project, execSpec -> {
+        project.exec(execSpec -> {
```

Comment:
The intention of LoggedExec was to not output the potentially very large output for the task if it succeeds. I'm not sure we want to lose that.